packages/models-library/src/models_library/services.py

```diff
@@ -286,6 +286,7 @@ class ServiceMetaData(ServiceCommonData):
     thumbnail: Optional[HttpUrl]
     description: Optional[str]
     classifiers: Optional[List[str]]
+    metadata: Optional[Dict[str, Any]]
```

Comment:
From the front-end I did a great step forward! On the other hand, the backend is totally blind to the metadata ... there is really a lot to do!

Reply:
I agree with @pcrespov here, the metadata field should at least have a validator for the metadata structure.

Reply:
@GitHK We will have to take over that part in the next iteration. My point here is that we should not give the impression this is completed.

Reply:
good to know!

Reply:
For now, the validation is only done in the frontend against this schema:
services/web/client/source/resource/form/service-metadata.json